torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -333,6 +401,16 @@ def aot_dispatch_autograd(
             fw_metadata.num_symints_saved_for_bw = len(symint_outs_saved_for_bw)
             inner_meta.num_symints_saved_for_bw = len(symint_outs_saved_for_bw)
             num_symints_saved_for_bw = len(symint_outs_saved_for_bw)
+            inner_meta.fw_donated_buffer = collect_donated_buffer_metadata(
+                fw_module,
+                bw_module,
+                num_inner_fwd_outputs,
+                num_fw_outs_saved_for_bw - num_symints_saved_for_bw,
+            )
+
+            if aot_config.enable_log:
+                msg = f"backward donated indices: {inner_meta.bw_donated_indices}"
```

Comment:
We actually already log the entire `fw_metadata` earlier in the AOTDispatcher life cycle - it's a bit annoying to have to accumulate more metadata here that we log separately.

Do you mind delaying the existing log so it happens after here, once all the metadata is set?